stunnel/stunnel.ml

```diff
@@ -159,6 +159,7 @@ let config_file verify_cert extended_diagnosis host port legacy =
   ; if extended_diagnosis then ["debug=4"] else []
   ; if verify_cert then
       ["verify=2"
+      ; sprintf "checkHost=%s" host
```

Comment:
This requires the new version of `stunnel`, right?
Please add a dependency in the specfile of `SPECS/ocaml-xen-api-libs-transitional.spec` to stunnel >= ..., something like: `Requires: stunnel >= 5.55`.

Reply:
You mean the specfile in xenserver-specs?
Sure. Will send another PR on that repo.